src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs

```diff
@@ -6,73 +6,36 @@
     {
         private const string Key = "metadata";
 
-        private readonly ILogWriter _writer;
         private readonly IMetricsLogger _metrics;
         private readonly IFunctionMetadataManager _metadataManager;
         private ConcurrentDictionary<BindingMetadata, string> _bindingMetricEventNames = new ConcurrentDictionary<BindingMetadata, string>();
 
         public FunctionInstanceLogger(
             IFunctionMetadataManager metadataManager,
             IMetricsLogger metrics,
-            IHostIdProvider hostIdProvider,
             IConfiguration configuration,
-            ILoggerFactory loggerFactory,
-            IDelegatingHandlerProvider delegatingHandlerProvider)
+            ILoggerFactory loggerFactory)
             : this(metadataManager, metrics)
         {
-            if (hostIdProvider == null)
-            {
-                throw new ArgumentNullException(nameof(hostIdProvider));
-            }
-
-            if (configuration == null)
-            {
-                throw new ArgumentNullException(nameof(configuration));
-            }
-
-            if (loggerFactory == null)
-            {
-                throw new ArgumentNullException(nameof(loggerFactory));
-            }
-
-            if (delegatingHandlerProvider == null)
-            {
-                throw new ArgumentNullException(nameof(delegatingHandlerProvider));
-            }
+            ArgumentNullException.ThrowIfNull(configuration, nameof(configuration));
+            ArgumentNullException.ThrowIfNull(loggerFactory, nameof(loggerFactory));
 
             string accountConnectionString = configuration.GetWebJobsConnectionString(ConnectionStringNames.Dashboard);
             if (accountConnectionString != null)
             {
-                CloudStorageAccount account = CloudStorageAccount.Parse(accountConnectionString);
-                var restConfig = new RestExecutorConfiguration { DelegatingHandler = delegatingHandlerProvider.Create() };
-                var tableClientConfig = new TableClientConfiguration { RestExecutorConfiguration = restConfig };
-
-                var client = new CloudTableClient(account.TableStorageUri, account.Credentials, tableClientConfig);
-                var tableProvider = LogFactory.NewLogTableProvider(client);
-
-                ILogger logger = loggerFactory.CreateLogger(ScriptConstants.LogCategoryHostGeneral);
-                logger.LogDebug("Azure WebJobs Dashboard is enabled");
-
-                string hostId = hostIdProvider.GetHostIdAsync(CancellationToken.None).GetAwaiter().GetResult() ?? "default";
-                string containerName = Environment.MachineName;
-                _writer = LogFactory.NewWriter(hostId, containerName, tableProvider, (e) => OnException(e, logger));
+                loggerFactory.CreateLogger<FunctionInstanceLogger>().LogWarning($"The {ConnectionStringNames.Dashboard} setting is no longer supported.");
```

Comment:
The issue says we wanted to also emit an aka.ms doc link here?

Reply:
Good catch -- added https://aka.ms/functions-dashboard. Currently points to `AzureWebJobsDashboard` docs, which we can update.